apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription.purs

```diff
@@ -29,7 +30,7 @@ subscription = make component
       { wrapperProgress: AsyncWrapper.Ready
       , pausedSubscriptions: Nothing
       , pendingAddressChanges: Nothing
-      , now: Nothing
+      , now: unsafeCoerce unit
```

Comment:
Is this a good practice? Cause it seems to me that if we do it here there would be no reason why we wouldn't do it in many other places